### PR TITLE
Abdullah week3 homework

### PR DESCRIPTION
**Pull request by heartwhite**

### CODE REVIEW

File: react-hyf/react-hyf-client/src/components/week3-homework.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      );
+  }
+  render() {
+    const { error, loaded, profiles, currentProfile } = this.state;
+    if (error) {
+      return <div>Error: {error.message}</div>;
+    } else if (!loaded) {
+      return <div>Loading...</div>;
+    } else {
+      console.log(this.state.profiles);
```
Comment (ArcoMul) [inline react-hyf/react-hyf-client/src/components/week3-homework.js]: Make sure to remove console.logs when creating a pull request ;-)

File: react-hyf/react-hyf-client/src/components/week3-homework.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+      return (
+        <div className="week3-homework">
+          {loaded && <ProfileImage url={profiles[currentProfile].photo} />}
```
Comment (ArcoMul) [inline react-hyf/react-hyf-client/src/components/week3-homework.js]: Above you are already checking if `loaded` is false or true. When this code is being executed `loaded` is always true, so no need to check that again here right?

File: react-hyf/react-hyf-client/src/components/previous-homeworks.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  };
+  render() {
+    const { display, text } = this.state;
+    return (
+      <div>
+        <div className={display}>
+          <PreviousHomework1 />
+          <PreviousHomework2 />
+        </div>
+        <button className="show-hide-button" onClick={this.changeClass}>
```
Comment (ArcoMul) [inline react-hyf/react-hyf-client/src/components/previous-homeworks.js]: Cool idea to hide the previous homework behind a button :-)

File: react-hyf/react-hyf-client/src/components/week3-homework.js (hunk 1 above)
Comment (heartwhite) [reply react-hyf/react-hyf-client/src/components/week3-homework.js]: I checked a few times. it is the last one I put again for another check. and I forgot. 
now removed.

File: react-hyf/react-hyf-client/src/components/week3-homework.js (hunk 2 above)
Comment (heartwhite) [reply react-hyf/react-hyf-client/src/components/week3-homework.js]: actually, I don't know at all how it works.  I mean the "&&" means what or how it works. my friend Talip (7alip > GitHub) showed that to me when I share my problem.
now I tried, it works with only first row. I removed other two check.
thanks

**Pull request closed without merging**